.github/workflows/deployment-releases.yml

```
@@ -0,0 +1,98 @@
+# Deployment workflow to be used for Jira reporting
+name: Deploy release
+
+on:
+  release:
+    types: [ published ]
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  deploy-release:
+    name: Deploy a release
+
+    # IMPORTANT: the workflow must have write access to deployments, otherwise the action will fail.
+    permissions:
+      deployments: write
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
```

Please update it here as well 
```suggestion
      - uses: actions/checkout@v3
```